runfolder/handlers.py

```diff
@@ -3,18 +3,31 @@
 from runfolder.services import *
 import tornado.web
 
+
 class BaseRunfolderHandler(BaseRestHandler):
     """Provides core logic for all runfolder handlers"""
+
+    def data_received(self, chunk):
+        """Empty implementation of abstract method"""
+        pass
```

Comment:
I'd go with `raise NotImplementedError` here - but I don't know what its the pythonic way of doing it.